Python/pystate.c

```diff
@@ -280,11 +280,19 @@ _PyInterpreterState_DeleteExceptMain()
 
     HEAD_LOCK();
     PyInterpreterState *interp = _PyRuntime.interpreters.head;
+    PyInterpreterState *tmp_interp = NULL;
     _PyRuntime.interpreters.head = NULL;
-    for (; interp != NULL; interp = interp->next) {
+    fprintf(stdout, "1. interp: %p\n", interp);
+    fprintf(stdout, "2. interp->next: %p\n", interp->next);
+    fprintf(stdout, "2.1. _PyRuntime.interpreters.main: %p\n", _PyRuntime.interpreters.main);
+    while(interp) {
+    //for (;interp != NULL;) { //interp=interp->next) {
```

Comment:
Please don't add commented code.

Reply:
yep, sure this PR is in marked WIP and I wanted to have your feedback about the code.